clicache/integration-test2/Cluster.cs

```diff
@@ -200,6 +204,7 @@ public int Start()
                     .withMaxHeap("256m")
                     .withJmxManagerPort(cluster_.jmxManagerPort)
                     .withJmxManagerStart(startJmxManager_)
+                    .withAllowAttach(allowAttach_, Address.address)
```

Comment:
What is behind the choice of the word "attach"? We are starting the debugger agent, why not use a name that suggests that, like `withDebugAgent(...)`.

Reply:
I like it! Good rename.